static/js/ui_init.js

```diff
@@ -209,6 +209,7 @@ function initialize_compose_box() {
                 : "End",
             narrow_to_compose_recipients_key_html:
                 (common.has_mac_keyboard() ? "⌘" : "Ctrl") + " + .",
+            max_message_length: page_params.max_message_length,
```

Comment:
Adding this would be a bug -- we currently allow message bodies to exceed the maximum length, with UI for indicating to the user how many characters they need to remove. 

Ideally, we'd do the same for the message edit UI; that might be a good follow-up project for you.

Reply:
Cool, Opened #22525.